dist/sharect.babel.js

```diff
@@ -79,6 +93,14 @@ var Sharect = function () {
       var DOCUMENT_SCROLL_TOP = window.pageXOffset || document.documentElement.scrollTop || document.body.scrollTop;
       _top = position.top + DOCUMENT_SCROLL_TOP - _iconSize - _arrowSize;
       _left = position.left + (position.width - _iconSize * _icons.length) / 2;
+      if (hasSelection()) {
```

Comment:
If selection's offsetParent is not `document.body`, adjust left and top positions with offsetParent positions